### PR TITLE
[16.0] [ADD] project_merge

### PR DESCRIPTION
**Pull request by ByteMeAsap**
This module adds a wizard to merge project tasks.

### CODE REVIEW

**Review by tarteo — APPROVED** [review]
Nice functionality!

File: project_merge/wizard/project_task_merge_views.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<odoo>
+        <record id="project_task_merge_view_form" model="ir.ui.view">
+            <field name='name'>project.task.merge.form</field>
+            <field name='model'>project.task.merge</field>
+            <field name='arch' type='xml'>
+                <form string='Merge Tasks'>
+                    <sheet>
+                        <group>
+                            <group>
+                                <field name="user_ids" widget="many2many_tags" />
```
Comment (tarteo) [inline project_merge/wizard/project_task_merge_views.xml]: Functional suggestion: Would be nice if the many2many_tags_avatar widget is used here.

File: project_merge/wizard/project_task_merge.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            "res_model": "project.task",
+            "views": [[False, "form"]],
+            "res_id": self.dst_task_id.id,
+        }
+
+    def merge_description(self):
+        return "<br/>".join(
+            self.task_ids.mapped(
+                lambda task: "Description from task <b>%s</b>:<br/>%s"
+                % (task.name, task.description or "No description")
```
Comment (tarteo) [inline project_merge/wizard/project_task_merge.py]: Functional suggestion: Don't add the description and "Description from task" if the description is empty

image

**Comment by tarteo** [comment]
@dreispt and @pedrobaeza can you please leave a review? :sweat_smile:

**Review by dreispt — APPROVED** [review]

File: project_merge/wizard/project_task_merge_views.xml (hunk 1 above)
Comment (ByteMeAsap) [reply project_merge/wizard/project_task_merge_views.xml]: Updated the widget

File: project_merge/wizard/project_task_merge.py (hunk 2 above)
Comment (ByteMeAsap) [reply project_merge/wizard/project_task_merge.py]: Updated the changes

**Pull request merged**